packages/amplify_auth_cognito/example/integration_test/sign_in_sign_out_test.dart

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:integration_test/integration_test.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:amplify_flutter/amplify.dart';
+import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
+import 'package:uuid/uuid.dart';
+
+import 'package:amplify_auth_cognito_example/amplifyconfiguration.dart';
+
+final uuid = Uuid();
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  final username = 'TEMP_USER-${uuid.v4()}';
+  final password = uuid.v4();
+
+  group('signIn and signOut', () {
+    setUpAll(() async {
+      if (!Amplify.isConfigured) {
+        final authPlugin = AmplifyAuthCognito();
+        await Amplify.addPlugins([authPlugin]);
+        await Amplify.configure(amplifyconfig);
+      }
+
+      await Amplify.Auth.signUp(
```

Comment:
How do these temporary users get removed?

Reply:
Bc our library has no way to delete users, we have to add a lambda to the test environment to periodically delete all the users. That can be done with the amplify CLI. I was not sure if that needed to be included in the README here bc it's not strictly necessary to run the tests.